services/api/Tweek.ApiService/MultiContext/MultiContextExtensions.cs

```diff
@@ -0,0 +1,61 @@
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.Design;
+using System.Linq;
+using System.Reflection;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.DependencyModel;
+using Tweek.ApiService.Addons;
+using Tweek.ApiService.Utils;
+using Tweek.Engine.Drivers.Context;
+
+namespace Tweek.ApiService.MultiContext
+{
+    public static class MultiContextExtensions
+    {
+        public static IEnumerable<IContextDriver> GetAllContextDrivers(this IServiceCollection services, IConfiguration configuration, IEnumerable<string> names)
```

Comment:
GetAllContextDrivers is misleading - it create side-effects of loading and running the addons code.
Additionally, it can return the mapping between addon-name to ContextDriver